x/crosschain/keeper/cctx_orchestrator_validate_outbound.go

```diff
@@ -240,7 +240,7 @@ func (k Keeper) processSuccessfulOutbound(
 	oldStatus := cctx.CctxStatus.Status
 	switch oldStatus {
 	case types.CctxStatus_PendingRevert:
-		cctx.SetReverted("", "revert executed")
+		cctx.SetReverted("", "")
```

Comment:
Consider adding the value to status message instead

Reply:
We already mention in the status message that we're in pending revert
In general we can review this whole workflow as part of this task: https://github.com/zeta-chain/node/issues/3194

Reply:
If in case there are any applications that use this string `revert executed` , they would need to reconfigure thier logic . 
Would keeping the string as part of the message be straightforward for now?